media/js/base/stat-counter.js

```diff
@@ -0,0 +1,11 @@
+var STATCOUNTER_ID = document.getElementsByTagName('html')[0].getAttribute('data-statcounter-id');
+
+// If doNotTrack is not enabled, it is ok to add StatCounter
+// @see https://bugzilla.mozilla.org/show_bug.cgi?id=1217896 for more details
+if (typeof window._dntEnabled === 'function' && !window._dntEnabled()) {
+    var sc_project=11297457;
+    var sc_invisible=1;
+    var sc_security=STATCOUNTER_ID;
+    var scJsHost = (("https:" == document.location.protocol) ? "https://secure." : "http://www.");
+    document.write("<script type='text/javascript' src='"+scJsHost+"statcounter.com/counter/counter.js'></script>");
```

Comment:
I know this may be in statcounter's original snippet, but `document.write` should generally be avoided for performance reasons. I'd suggest using `createElement` like we do for [other third party snippets](https://github.com/mozilla/bedrock/blob/master/media/js/base/optimizely-snippet.js)?